pkg/console/subresource/consoleserver/types.go

```diff
@@ -66,6 +67,12 @@ type ClusterInfo struct {
 	ControlPlaneToplogy configv1.TopologyMode `yaml:"controlPlaneTopology,omitempty"`
 }
 
+// Monitoring holds configuration for monitoring related services
+type MonitoringInfo struct {
```

Comment:
noticed that [types.go](https://github.com/openshift/console/pull/11712/files#diff-4d2abe7485c7a156905e0fc628a0edf2aaed9ce1a6d0dda5cc7df661fe27d529) already contain some field.. we should have field parity between console and console-operator

Reply:
Correct, we removed the other fields in https://github.com/openshift/console-operator/pull/562 but we never propagate the change to the console code base (https://github.com/openshift/console/pull/9276 was closed on the assumption that the monitoring URLs could be useful eventually for the multi-cluster console story).

Not sure which way we want to go but I see several options:
* Add back the fields in the console-operator project in this PR.
* Remove the fields from the console project as part of https://github.com/openshift/console/pull/11712.
* Resolve the situation once 4.12 opens.

Let me know what you think but I'm happy to take care of it :)

Reply:
@simonpasquier if we need those fields either in console or console-operator, they should be present in both api, so they match. Since I think those fields are still needed we should bring them back part of this PR